src/Resources/app/administration/src/module/swag-migration/snippet/de-DE.json

```
@@ -364,6 +364,7 @@
         "flowChart": {
           "premapping": "Prüfen",
           "fetchData": "Lesen",
+          "applyFixes": "Applying fixes",
```

We should translate these, before we forget to do so later 😅 

```suggestion
          "applyFixes": "Fehler beheben",
```